pkg/logger/logger.go

```diff
@@ -117,11 +119,21 @@ func zapSugaredLoggerTo(options *Opts) *zap.SugaredLogger {
 
 	if options.Console {
 		encCfg := zap.NewDevelopmentEncoderConfig()
-		encCfg.LevelKey = ""
-		encCfg.TimeKey = ""
-		encCfg.NameKey = ""
-		encCfg.CallerKey = ""
-		enc = zapcore.NewConsoleEncoder(encCfg)
+		if options.OutputFormat == "json" {
+			encCfg.CallerKey = util.GetOSEnv("CALLER_KEY", "caller")
```

Comment:
Are those variables documented somewhere? If not, could you please add a note in the Kogito CLI README section?

Reply:
maybe point to the zap logger docs?

Reply:
@ricardozanini , I've added a section to the readme.
@spolti Our use of the zap logger isn't exposed to users, so I'm not sure whey we'd do that?